app/views/tag_wranglers/show.html.erb

```diff
@@ -15,8 +15,14 @@
     <h3 class="heading"><%= ts('Assigned Fandoms') %></h3>
     <% if @fandoms.empty? %>
       <p><%= ts("Your lack of fandoms makes the wrangulator sad. Fandoms can be assigned via the") %> <%= link_to ts('tag wranglers page'), tag_wranglers_path %>.</p>
+      <% if (@wrangler == @current_user || logged_in_as_admin?) && @wrangler.last_wrangling_activity %>
+        <p><%= t(".last_wrangled", wrangler_login: @wrangler.login) %> <span class="datetime"><%= time_in_zone(@wrangler.last_wrangling_activity.updated_at) %></span>.</p>
```

Comment:
Two extra things I just noticed:
1. In the preview screenshot, it looks like there's an extra space between the time and the period after it. That comes from `time_in_zone`, which includes an extra space at the end in some (but not all) cases. It might be good to strip the extra space, either here or in `time_in_zone`.
2. Now that you're using `t()` instead of `ts()`, it would probably be good to follow the [internationalization standards](https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards), which specify that the whole sentence should be a single translatable string (in case there are languages where the positions of the interpolated variables in the sentence differ from English).

   In other words, it'd be good to define the string as `"%{wrangler_login} last wrangled at %{time}."`, and pass `tag.span(time_in_zone(@wrangler.last_wrangling_activity.updated_at), class: "datetime")` as an extra param.

   (To prevent the HTML from being escaped, you might need to change the key name to end in `_html`. The alternative would be manually marking the string as `html_safe`, but I'm not sure how Brakeman would feel about that.)

   Alternatively, you can switch back to `ts()` and leave it for whoever ends up internationalizing this view.

Reply:
I ended up removing the extra space in `time_in_zone` to avoid some annoying chaining with `.strip.html_safe`. I checked the usage in templates and that seems safe, plus the attached screenshot.

![existing_time_in_zone](https://user-images.githubusercontent.com/13002992/188712359-6505abcf-e2c4-481e-af1a-3a5513b70eb8.png)

Thanks for the tips and reminder! I've updated the template, sticking with `t()` since I've been asked to switch on similar changes in the past.

Reply:
Did you check it with challenge dates? The one case where I'm concerned that space might be necessary is when viewing challenge dates logged in as a user whose time zone doesn't match the challenge's.

Reply:
Right you are! I've re-shuffled things a bit to make sure that works as well, here's a screenshot after those changes.

![challenge with closing date in a different time zone](https://user-images.githubusercontent.com/13002992/188747050-e0c2269a-3c56-44de-997c-f3a2b16c1fbb.png)

Thanks for keeping me honest!